lib/matplotlib/testing/compare.py

```diff
@@ -146,16 +146,15 @@ def __call__(self, orig, dest):
             weakref.finalize(self._tmpdir, self.__del__)
         if (not self._proc  # First run.
                 or self._proc.poll() is not None):  # Inkscape terminated.
+            if self._proc is not None and self._proc.poll() is not None:
+                for stream in filter(None, [self._proc.stdin,
+                                            self._proc.stdout,
+                                            self._proc.stderr]):
+                    stream.close()
```

Comment:
Not really a problem, but is it really possible for the process' standard streams to still be open even though the process has terminated? (`poll() is not None`)